src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/HandlerFactory.java

```diff
@@ -74,6 +74,8 @@ public HandlerFactory() {
         register(TokenTypes.INDEX_OP, IndexHandler.class);
         register(TokenTypes.LITERAL_SYNCHRONIZED, SynchronizedHandler.class);
         register(TokenTypes.LAMBDA, LambdaHandler.class);
+        register(TokenTypes.ANNOTATION_DEF, ClassDefHandler.class);
+        register(TokenTypes.ANNOTATION_FIELD_DEF, AnnotationFieldDefHandler.class);
```

Comment:
@rnveach 
It would actually be possible to get rid of `AnnotationFieldDefHandler`  completely and simply just edit the `checkIndentation` method of `MemberDefHandler` (at [L69](https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/MemberDefHandler.java#L69) and [L56](https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/MemberDefHandler.java#L56) ), unless future additions/changes are required.

I will share my results on the reports soon. Hopefully a better decision can be taken based on the report